docker/drone/src/main/java/org/arquillian/cube/docker/drone/VideoConversionLifecycleManager.java

```diff
@@ -19,13 +19,16 @@
     @Inject
     Instance<SeleniumContainers> seleniumContainersInstance;
 
-    public void startConversion(@Observes AfterSuite afterSuite, CubeRegistry cubeRegistry) {
-
-        initConversionCube(cubeRegistry);
-        flv2mp4.create();
-        flv2mp4.start();
-
-        afterConversionEvent.fire(new AfterConversion());
+    public void startConversion(@Observes AfterSuite afterSuite, CubeDroneConfiguration cubeDroneConfiguration,
+        CubeRegistry cubeRegistry) {
+
+        if (cubeDroneConfiguration.isRecording()) {
```

Comment:
Alternatively, you could reverse an if and `return;`  but that's more a matter of style.

Reply:
I chose this way to be consistent with VncRecorderLifecycleManager where the check is already implemented :)